data/mods/Aftershock/suit_operating_time.md

```diff
@@ -0,0 +1,106 @@
+ |-----------------------|-------------------------------------------------------------------------|
+ | 1hour per every 5 enc | Count only the most encumbered part                                     |
+ |        1 hours        | Less than 40 armor against all hazards                                  |
+ |        1 hours        | Less than 20 armor against all hazards                                  |
+ |        6 hours        | Less than 10 armor against all hazards                                  |
+ |        2 hours        | Has the fragile tag or is made from a very fragile material (ex: glass) |
+ |        4 hours        | Less than 90% coverage in the most encumbered part                      |
+ |        4 hours        | No protection against gases                                             |
+ |        12 hours       | No protection against the cold                                          |
+ |        2 hours        | No protection against other environmental factors (ex: acid, electricity, radiation)  |
+ |        4 hours        | Gear doesn't give passive bonuses to physical attributes(ex:dex or str), skills (ex:athletics) or wont grant physical based proficiencies|
+ |        4 hours        | Doesn't grant any combat/movement related spells or enchantments         |
+ |        4 hours        | Gear doesn't give passive bonuses to mental attributes (per or int), skills (ex: applied science) or wont grant knowledge based proficiencies |
+ |        1 hours        | Doesn't grant any mental/knowledge related spells or enchantments        |
+ |        2 hours        | Has a non swappable integral battery.                                   |
+ |        2 hours        | Wont grant night/heat-vision or clairvoyance effects.                   |
+
+Once you have determined the operating time using the table above, simply divide the number 277777.78 by the indicated number of hours to obtain your new suit's `power_draw` json value.
+
+For player convenience, the power draw and battery capacity of all clothing pieces that complete a single matching outfit (for example: The magellan exosuit and its helmet) should be the same. 
+
+#### Example Calculation
+
+Consider the Magellan Exosuit, which has the following definitions:
+```
+  {
+    "id": "afs_magellan_suit",
+    "type": "TOOL_ARMOR",
+    "category": "armor",
+    "name": { "str": "Magellan exosuit" },
+    "description": "A high-quality, civilian grade EVA suit often employed by well-established frontier research and exploration associations.  Designed to support the exploration of challenging terrain, it offers respectable protection against common environmental hazards like extreme temperatures, inhospitable atmospheres, and  light radiation.  It leaves arms and hands relatively unencumbered to facilitate the manipulation of scientific instruments.\n\nAn integral battery allows the suit to operate for up to 36 hours, but complicates on the field recharging",
```

Comment:
Change as needed to match the actual definition, of course.

Reply:
Thanks changed some of the calculation numbers and forgot to change the result here